### DISCUSSION TITLE
Problem with Plugins

### DISCUSSION THREAD

**Discussion by Tonaion02** [bevyengine/bevy#14738] [Q&A]
I am currently try to understand what plugins i must add to run a Window and Render something, my Cargo.toml is the sequent:
![image](https://github.com/user-attachments/assets/da21b6f8-a5b3-4832-9638-81d249622077)
The plugins that i added are these:
![image](https://github.com/user-attachments/assets/8050a17f-abdb-49d0-9dd5-10734a130ef4)
And i get a strange error:
![image](https://github.com/user-attachments/assets/831b5d88-2ef4-4f0d-a63a-f28c79973f3f)
Someone can help me?

**Comment by alice-i-cecile** [comment]
I don't know how to solve your error unfortunately, but can you copy-paste the error that you received into your text? That'll make it much easier to search for :)

**Comment by nuzzles** [comment]
I believe the ordering of your plugins is at fault here.

Ensure the strict ordering of DefaultPlugins (at least the ones you need).

You can find the list ordering here: https://docs.rs/bevy/latest/bevy/struct.DefaultPlugins.html
